### ISSUE TITLE
Multiple-Document Interface

### ISSUE THREAD

**Issue by c-sanchez** [bgrabitmap/lazpaint#101]
Allow open and edit different images in the same instance
For now we can edit just one image/project at time in the same instance of LazPaint

![lazpaint-documents](https://i.ibb.co/xmyLZKW/lazpaint-documents.png)
![phoxo-documents](https://i.ibb.co/8MtCsHb/phoxo-documents.png)
![pictbear-documents](https://i.ibb.co/XLCp2pr/pictbear-documents.png)

**Comment by lainz** [comment]
I like the idea but I prefer "Tabs" like a browser does.

**Comment by c-sanchez** [comment]
yeah, tabs is the idea for images in "full screen", in both Phoxo and PictBear it's possible, check they have minimize/maximize/close buttons.
in pictbear we have the tabs, however in both it's possible change tab with "ctrl+tab", like browser heh.

**Comment by lainz** [comment]
That's final fantasy (the pictures)?

**Comment by c-sanchez** [comment]
Heh nop, is Castlevania Symphony of the night, are Alucard, Richter and Maria :)

**Comment by lainz** [comment]
Ok, I just played some SNES or GBA of Castlevania, not the PSX one. But I can't remember the names.

**Comment by c-sanchez** [comment]
From GBA I've played all, Circle of the moon, Harmony of dissonance and Aria of sorrow this probably the "spiritual sucessor" of Symphony of the night and one of the best in the franchise.

The art style of Ayami Kojima's is cool, however in Castlevania games for NDS was used a anime style instead.
Maybe you find interesting this topic :P
http://www.romhacking.net/forum/index.php?topic=22445.0

**Comment by circular17** [comment]
That's a possible enhancement indeed. That would need lots of refactoring I suppose.

**Comment by circular17** [comment]
Interesting
